.github/workflows/packages.yaml

```diff
@@ -10,8 +10,40 @@ on:
   workflow_dispatch:
 
 jobs:
-  build-distro-packages:
-    name: build packages
+  build-distro-packages-arm64:
+    runs-on: ubuntu-latest
+    name: build arm64 packages
+    strategy:
+      fail-fast: true
+      matrix:
+        format: [ rpm, deb ]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: uraimo/run-on-arch-action@v2.1.1
+        name: Build the ${{matrix.format}} packages
+        with:
+          arch: aarch64
+          distro: ubuntu20.04
+          run: |
+            apt-get update && \
+            apt-get install -y --no-install-recommends \
+            build-essential \
+            cmake \
+            file \
+            rpm \
+            make 
+            cmake .
+            echo ${{ matrix.format }} | awk '{print toupper($0)}' | xargs -I{} cpack -G {}
+
+      - name: Store the master package artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.format }}
```

Comment:
Is this going to collide with the artefact name for amd so we only get one of them?

Reply:
-rw-rw-r--@ 1 niedbalski  staff   70482 Mar  3 10:59 cmetrics_0.3.0_arm64-headers.deb
-rw-rw-r--@ 1 niedbalski  staff  139142 Mar  3 10:59 cmetrics_0.3.0_arm64.deb
-rw-rw-r--@ 1 niedbalski  staff   70478 Mar  3 10:55 cmetrics_0.3.0_amd64-headers.deb
-rw-rw-r--@ 1 niedbalski  staff  151548 Mar  3 10:55 cmetrics_0.3.0_amd64.deb

Reply:
Yeah but the name of the uploaded artefact will be `rpm` or `deb` which then collides as it does not have arch

Reply:
See https://github.com/calyptia/cmetrics/actions/runs/1924225317